### PR TITLE
Add iw package

### PR DESCRIPTION
**Pull request by cheat-sc**

### CODE REVIEW

**Comment by masami256** [comment]
@cheat-sc  It looks tab and space are mixed.  Could you check indent please?

```
masami@dev:~/projects/emlinux/meta-debian-extended (test56)$ git am ../56.patch
Applying: iw: Add iw package
.git/rebase-apply/patch:29: space before tab in indent.
                $(wildcard .git/index .git/refs/tags)
.git/rebase-apply/patch:30: space before tab in indent.
        @$(NQ) ' GEN ' $@
.git/rebase-apply/patch:35: space before tab in indent.
        @$(NQ) ' CC  ' $@
warning: 3 lines add whitespace errors.
```

File: recipes-debian/iw/iw_debian.bb [hunk 1]
```diff
@@ -0,0 +1,40 @@
+# base recipe: meta/recipes-connectivity/iw/iw_4.14.b
```
Comment (hiraku-wfs) [inline recipes-debian/iw/iw_debian.bb]: In this case, you should refer iw_5.0.1.bb at poky master.

File: recipes-debian/iw/files/fix-build-error.patch [hunk 2]
```diff
@@ -0,0 +1,25 @@
+--- a/Makefile	2019-02-04 22:37:44.000000000 +0900
```
Comment (hiraku-wfs) [inline recipes-debian/iw/files/fix-build-error.patch]: This fix is almost same as separate-objdir.patch in iw_5.0.1.bb at poky master.
Can we copy the patch into meta-debian-extended as it is?

**Comment by hiraku-wfs** [comment]
> @cheat-sc It looks tab and space are mixed. Could you check indent please?
> 
> ```
> masami@dev:~/projects/emlinux/meta-debian-extended (test56)$ git am ../56.patch
> Applying: iw: Add iw package
> .git/rebase-apply/patch:29: space before tab in indent.
>                 $(wildcard .git/index .git/refs/tags)
> .git/rebase-apply/patch:30: space before tab in indent.
>         @$(NQ) ' GEN ' $@
> .git/rebase-apply/patch:35: space before tab in indent.
>         @$(NQ) ' CC  ' $@
> warning: 3 lines add whitespace errors.
> ```

@masami256 This is because it's "patch to patch". So I think this is OK, isn't it?

File: recipes-debian/iw/files/fix-build-error.patch (hunk 2 above)
Comment (hiraku-wfs) [reply recipes-debian/iw/files/fix-build-error.patch]: If you need to modify the patch, please prepend a reference like below.
https://github.com/meta-debian/meta-debian/blob/warrior/recipes-debian/pciutils/pciutils/configure.patch

File: recipes-debian/iw/iw_debian.bb (hunk 1 above)
Comment (cheat-sc) [reply recipes-debian/iw/iw_debian.bb]: I have change to use the recipe on the master branch.

File: recipes-debian/iw/files/fix-build-error.patch (hunk 2 above)
Comment (cheat-sc) [reply recipes-debian/iw/files/fix-build-error.patch]: I have copied the file from the master branch. I could apply it without any modifications.

**Comment by cheat-sc** [comment]
I have pushed a commit including those fixes. Please review it again

**Comment by masami256** [comment]
@hiraku-wfs 

> @masami256 This is because it's "patch to patch". So I think this is OK, isn't it?

yes, that's ok.

**Comment by hiraku-wfs** [comment]
LGTM.

@masami256 If it's fine with you, please merge this PR.

**Pull request merged**